tests/test_skill_lab.py

```
@@ -74,6 +73,83 @@
+        "category": "conditioning",
+        "difficulty": "beginner",
+        "required_equipment": "ball",
+        "duration_min": 5,
+    },
+]
+
+
+def _fake_query_drills(query_texts, n_results=3, where=None):
+    """Return filtered drill results based on where clause."""
+    docs, metas = [], []
+    for d, m in zip(_FAKE_DRILL_DOCS, _FAKE_DRILL_METAS):
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)